src/workerd/api/node/util.h

```diff
@@ -251,6 +267,7 @@ class UtilModule final: public jsg::Object {
 
     JSG_METHOD(getBuiltinModule);
     JSG_METHOD(processExitImpl);
+    JSG_READONLY_INSTANCE_PROPERTY(processPlatform, getProcessPlatform);
```

Comment:
One last nit.. since the value wouldn't be expected to change, let's make this `JSG_LAZY_READONLY_INSTANCE_PROPERTY` so the getter is only ever called once.